src/components/CheckEditor/CheckProbes/PrivateProbesAlert.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { Alert, LinkButton, Stack } from '@grafana/ui';
+
+import { ROUTES } from 'types';
+import { getRoute } from 'components/Routing.utils';
+
+export const PrivateProbesAlert = () => {
+  const [display, setDisplay] = useState(true);
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <Alert
+      title="You haven't set up any private probes yet."
+      severity="info"
+      onRemove={() => {
+        setDisplay(false);
+      }}
+    >
+      <Stack gap={1} direction={'column'} alignItems={'flex-start'}>
+        Probes are automated tools that test websites and apps. They act like users by sending requests and checking the
+        responses.
```

Comment:
Let's update the copy here as it isn't too specific to private probes but probes on a whole. @heitortsergent any thoughts?

Reply:
Changed it to the following, based on the text from the docs (https://grafana.com/docs/grafana-cloud/testing/synthetic-monitoring/set-up/set-up-private-probes/#set-up-private-probes)

![image](https://github.com/user-attachments/assets/7bb462e1-bd6c-4b95-99d4-00dfc4bd092f)